komodo/check_up_to_date_pypi.py

```diff
@@ -147,9 +148,14 @@ def run_check_up_to_date(
         f"{sys.version_info.micro}"
     ),
     propose_upgrade=False,
+    ignore=None,
 ):
     yaml = yaml_parser()
-    releases = load_from_file(yaml, release_file)
+    releases: dict = load_from_file(yaml, release_file)
+    if ignore:
+        for package_name, package_version in list(releases.items()):
+            if re.search(ignore, f"{package_name} {package_version}"):
```

Comment:
What if we have some pypi package that includes "main" in its package name? We should probably be explicit whether the regexp refer to the version or name.